webpack/components/ProxmoxContainer/ProxmoxContainerNetwork.js

```diff
@@ -0,0 +1,169 @@
+        value: `net${netId}`,
+      },
+      name: {
+        name: `${paramScope}[interfaces_attributes][${nextId}][name]`,
+        value: `eth${netId}`,
+      },
+      bridge: {
+        name: `${paramScope}[interfaces_attributes][${nextId}][bridge]`,
+        value: bridges?.[0]?.iface || '',
+      },
+      dhcp: {
+        name: `${paramScope}[interfaces_attributes][${nextId}][dhcp]`,
+        value: '0',
+      },
+      cidr: {
+        name: `${paramScope}[interfaces_attributes][${nextId}][cidr]`,
+        value: '',
+      },
+      gw: {
+        name: `${paramScope}[interfaces_attributes][${nextId}][gw]`,
+        value: '',
+      },
+      dhcp6: {
+        name: `${paramScope}[interfaces_attributes][${nextId}][dhcp6]`,
+        value: '0',
+      },
+      cidr6: {
+        name: `${paramScope}[interfaces_attributes][${nextId}][cidr6]`,
+        value: '',
+      },
+      gw6: {
+        name: `${paramScope}[interfaces_attributes][${nextId}][gw6]`,
+        value: '',
+      },
+      tag: {
+        name: `${paramScope}[interfaces_attributes][${nextId}][tag]`,
+        value: '',
+      },
+      rate: {
+        name: `${paramScope}[interfaces_attributes][${nextId}][rate]`,
+        value: '',
+      },
+      firewall: {
+        name: `${paramScope}[interfaces_attributes][${nextId}][firewall]`,
+        value: '0',
+      },
+    };
+    setNextId(prevId => {
+      if (availableIds.length > 0) {
+        setAvailableIds(availableIds.slice(1));
+      } else {
+        prevId += 1;
+      }
+      setUsedIds(prevIds => new Set(prevIds).add(netId));
+      const newId = availableIds.length > 0 ? availableIds[0] : prevId;
+      const newInterface = {
+        id: newId,
+        bridges,
+        data: initData,
+        networks: network,
+      };
+
+      setInterfaces(interfaces => [...interfaces, newInterface]);
+      return prevId;
+    });
+  };
+
+  const removeInterface = idToRemove => {
+    const newInterfaces = interfaces.filter(nic => nic.id !== idToRemove);
+    setInterfaces(newInterfaces);
+    setAvailableIds([...availableIds, idToRemove].sort((a, b) => a - b));
+    setUsedIds(prevIds => {
+      const newIds = new Set(prevIds);
+      newIds.delete(idToRemove);
+      return newIds;
+    });
+  };
+
+  const updateNetworkData = (id, updatedData) => {
+    setInterfaces(
+      interfaces.map(net =>
+        net.id === id ? { ...net, data: updatedData } : net
+      )
+    );
+  };
+
+  return (
+    <div>
+      <PageSection padding={{ default: 'noPadding' }}>
+        <Button onClick={addInterface} variant="secondary">
+          {__('Add Interface')}
+        </Button>
+        {interfaces.map(nic => (
+          <div key={nic.id} style={{ position: 'relative' }}>
+            <div
+              style={{
+                marginTop: '10px',
+                display: 'flex',
+                justifyContent: 'space-between',
+                alignItems: 'center',
+              }}
```

Comment:
Here is also a lot of style that can move to a css file.

Reply:
will be part of another PR.